locale/en/blog/announcements/nodejs16-eol.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2022-06-08T16:00:00Z
+category: Announcements
+title: Bringing forward the End-of-Life Date for Node.js 16
+slug: nodejs16-eol
+layout: blog-post.hbs
+author: Richard Lau
+---
+
+# Bringing forward the End-of-Life Date for Node.js 16 to September 11th, 2023
```

Comment:
This is super-awkward and wordy. "Bringing forward" makes it sound like we're revealing what the date is for the first time or something like that. It does not clearly indicate that we are reducing the support lifetime, although I imagine that is the intent.

```suggestion
# Changing the End-of-Life Date for Node.js 16 to September 11th, 2023
```

Reply:
Updated.

Reply:
> Doing some more Google searches, it looks like maybe "bringing forward" is common in UK English and, pretty much, English throughout the world except in the US (where it is more common to see "move forward"). I'm OK if this stays as "Bringing forward". Non-blocking, but I do think I prefer "Changing".

I've already changed it, so I'll leave it changed 🙂. It's hard to spot English phrases that don't travel well, so anything that helps clarify the message to a wider audience is okay with me. I've even rewritten all of the dates in the post to the doesn't-look-correct-to-me-from-the-UK "month date, year" format 😂.